Release notes for 8.0.0-rc2 (#13669)
elastic/logstash: docs/static/releasenotes.asciidoc

This section summarizes the changes in the following releases:

* <<logstash-8-0-0-rc2,Logstash 8.0.0-rc2>>
* <<logstash-8-0-0-rc1,Logstash 8.0.0-rc1>>
* <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
* <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-0-0-rc2]]
=== Logstash 8.0.0-rc2 Release Notes

> ==== Notable issues fixed

Reviewer: ```suggestion
[[notable-8.0.0-rc2]]
==== Notable issues fixed
```

> This release is mainly a consolidation of the code base.

Reviewer: I don't think this sentence has much meaning for a user. I think we can take it out.

> Fixed long-standing issue with event's count exposed in metrics when the events are dropped from pipelines https://github.com/elastic/logstash/pull/13593[#13593].
> Mitigates a memory leak that can be caused by using UUIDs or other high-cardinality field names https://github.com/elastic/logstash/pull/13642[#13642].
> Fixed an issue with Azure input plugin, that when used in a pipeline made Logstash to crash https://github.com/elastic/logstash/pull/13603[#13603].

Reviewer: ```suggestion
* Fixed long-standing issue in which the `events.out` count incorrectly included events that had been dropped with the drop filter.  
Now the total out event count includes only events that reach the out stage. https://github.com/elastic/logstash/pull/13593[#13593].
* Reduced scope and impact of a memory leak that can be caused by using UUIDs or other high-cardinality field names https://github.com/elastic/logstash/pull/13642[#13642].
* Fixed an issue with the Azure input plugin that caused Logstash to crash when it was used in a pipeline. https://github.com/elastic/logstash/pull/13603[#13603].
```

This section summarizes the changes in the following releases:

* <<logstash-8-0-0-rc2,Logstash 8.0.0-rc2>>
* <<logstash-8-0-0-rc1,Logstash 8.0.0-rc1>>
* <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
* <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-0-0-rc2]]
=== Logstash 8.0.0-rc2 Release Notes

[[notable-8.0.0-rc2]]
==== Notable issues fixed

> * Fixed long-standing issue in which the `events.out` count incorrectly included events that had been dropped with the drop filter.
> Now the total out event count includes only events that reach the out stage. https://github.com/elastic/logstash/pull/13593[#13593].
> * Reduced scope and impact of a memory leak that can be caused by using UUIDs or other high-cardinality field names https://github.com/elastic/logstash/pull/13642[#13642].
> * Fixed an issue with the Azure input plugin that caused Logstash to crash when it was used in a pipeline. https://github.com/elastic/logstash/pull/13603[#13603].

Reviewer: ```suggestion
* Fixed long-standing issue in which the `events.out` count incorrectly included events that had been dropped with the drop filter.
Now the total out event count includes only events that reach the out stage. https://github.com/elastic/logstash/pull/13593[#13593]
* Reduced scope and impact of a memory leak that can be caused by using UUIDs or other high-cardinality field names https://github.com/elastic/logstash/pull/13642[#13642]
* Fixed an issue with the Azure input plugin that caused Logstash to crash when the input was used in a pipeline. https://github.com/elastic/logstash/pull/13603[#13603]
```

This section summarizes the changes in the following releases:

> * <<logstash-8-0-0-rc1,Logstash 8.0.0-rc2>>

Reviewer: ```suggestion
* <<logstash-8-0-0-rc2,Logstash 8.0.0-rc2>>
```
